Update Book to 0.8

cart opened the pull request:
Just bump some version numbers

superdump reviewed (approved):
This looks fine but I can’t easily check for more from my phone. :)

cart commented:
Skipped bors for faser deploy ... the blog post failed to deploy due to the now removed rect example (which i just fixed in this pr).
Non-standard, but time is precious :)
